x-pack/test/functional/apps/index_management/home_page.ts

```diff
@@ -82,9 +81,10 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
         const url = await browser.getCurrentUrl();
         expect(url).to.contain(`/component_templates`);
 
-        // There should be no component templates by default, so we verify the empty prompt displays
-        const componentTemplateEmptyPrompt = await testSubjects.exists('emptyList');
-        expect(componentTemplateEmptyPrompt).to.be(true);
+        // Verify content. Component templates may have been created by other apps, e.g. Ingest Manager,
+        // so we don't make any assertion about the presence or absence of component templates.
+        const componentTemplateList = await testSubjects.exists('componentTemplateList');
```

Comment:
If no components are created automatically then this test would fail, right? I am not sure that we need this test, as this use case is covered by our component integration tests (mocked empty list = empty prompt, results = template list).